Source/WebInspectorUI/UserInterface/Views/TabBrowser.js

```diff
@@ -47,6 +47,8 @@ WI.TabBrowser = class TabBrowser extends WI.View
             this._detailsSidebar.addEventListener(WI.Sidebar.Event.WidthDidChange, this._handleSidebarWidthDidChange, this);
             this._detailsSidebar.addEventListener(WI.Sidebar.Event.SidebarPanelSelected, this._handleSidebarPanelSelected, this);
             this._detailsSidebar.addEventListener(WI.MultiSidebar.Event.SidebarAdded, this._handleMultiSidebarSidebarAdded, this);
+            this._detailsSidebar.addEventListener(WI.MultiSidebar.Event.HeightDidChange, this._handleSidebarWidthDidChange, this);
```

Comment:
Did the latter.

Reply:
> Did the latter.

This may have gotten rewritten in between patch updates. The handler is still `this._handleSidebarWidthDidChange` when you probably wanted to rename it to `this._handleSidebarSizeDidChange`.

Reply:
Yeah, it didn't end up in the commit. It should be fixed now though.